packages/core/src/utils/Target.ts

```diff
@@ -0,0 +1,19 @@
+import { DebugEvent } from '../types';
+
+type EventListener = (e: DebugEvent) => void;
+
+export class Target {
+  private listeners: EventListener[] = [];
```

Comment:
If we support it, could it make sense to use private class fields instead?

Reply:
I thought this was a private class field?

Reply:
Sorry I shoulda clarified - JS private class fields with `#`. Just had a look and they don't seem to be supported in Firefox yet. Feel free to resolve

Reply:
TIL!